src/components/Scheme/ContributionRewardExtRewarders/Competition/List.tsx

```diff
@@ -104,8 +107,45 @@ export default withSubscription({
   loadingComponent: null,
   errorComponent: (props) => <div>{ props.error.message }</div>,
   checkForUpdate: [],
-  createObservable: async (_props: IExternalProps) => {
-    return primeCacheForSubmissionsAndVotes();
+  createObservable: async (props: IExternalProps) => {
+    // prime the cache before creating the observable...
+    const cacheQuery = gql`query cacheSuggestions {
+      proposals (where: {scheme: "${props.scheme.id}"}) {
+        id
+        competition {
+          id
+      endTime
+      contract
+      suggestionsEndTime
+      createdAt
+      numberOfVotesPerVoters
+      numberOfWinners
+      rewardSplit
+      snapshotBlock
+      startTime
+          suggestions {
+            ...CompetitionSuggestionFields
+            }
+          votes { 
+            ...CompetitionVoteFields
+          }
+        }
+      }
+    }
+    ${CompetitionSuggestion.fragments.CompetitionSuggestionFields}
+    ${CompetitionVote.fragments.CompetitionVoteFields}
+    `;
+
+    const arc = await getArc();
+    await arc.sendQuery(cacheQuery, {subscribe: true});
+    // end cache priming
+
+    return combineLatest(
```

Comment:
You can perhaps just return `of()`.  The List component doesn't expect anything.